opencti-platform/opencti-front/src/private/components/analysis/external_references/ExternalReferenceCreation.tsx

```diff
@@ -421,12 +431,30 @@ ExternalReferenceCreationProps
                       }}
                       InputLabelProps={{ fullWidth: true, variant: 'standard' }}
                       InputProps={{
-                        fullWidth: true,
-                        variant: 'standard',
+                        classes: {
+                          fullWidth: true,
+                          variant: 'standard',
+                        },
+                        onChange: (event: React.ChangeEvent<HTMLInputElement>) => {
+                          const fileName = event.target.value.split('\\').pop();
+                          const externalIdValue = (document.getElementById('external_id') as HTMLInputElement).value;
+                          if (!externalIdValue && fileName) {
+                            setFieldValue('external_id', fileName);
+                          }
+                        },
                       }}
                       fullWidth={true}
                     />
                   )}
+                  <Field
+                    component={MarkDownField}
+                    name="description"
+                    label={t('Description')}
+                    fullWidth={true}
+                    multiline={true}
+                    rows="4"
+                    style={{ marginTop: 20, marginBottom: 20 }}
```

Comment:
Could this be a class ?

Reply:
<Field> doesn't seem to support classes= or className=  --> to be seen together